### PR TITLE
[IMP][8.0] website_product_supplier: some improvements.

### PR DESCRIPTION
**Pull request by sergio-teruel**

### CODE REVIEW

**Comment by rafaelbn** [comment]
Why travis fails?

**Comment by yajo** [comment]
Remove the pot.

File: website_product_supplier/views/product_supplier_view.xml [hunk 1]
```diff
@@ -162,21 +162,33 @@
...
+        <div class="col-md-6 col-md-push-6">
+            <div class="row mt16 mb16">
+                <div id="actions" class="col-md-8 pull-right">
+                    <t t-call="website_product_supplier.actions"/>
+                </div>
             </div>
-            <div id="actions" class="col-md-2 col-xs-4">
-                <t t-call="website_product_supplier.actions"/>
+        </div>
+        <div class="col-md-6  col-md-pull-6">
```
Comment (yajo) [inline website_product_supplier/views/product_supplier_view.xml]: double space.

File: website_product_supplier/views/product_supplier_view.xml [hunk 2]
```diff
@@ -220,19 +234,29 @@
...
-                <span class="label label-info"><i class="fa fa-fw fa-lock"/>Not Published</span>
+                <i class="fa fa-close"/>
             </span>
         </td>
-        <td>
-            <button class="btn btn-default btn-del-product pull-right" type="button"
-                    t-att-oe_model_id="supplierinfo.id">
-                <i class="fa fa-trash"></i>
-            </button>
+        <td name="product_edit" class="text-center">
```
Comment (yajo) [inline website_product_supplier/views/product_supplier_view.xml]: not name, id.

File: website_product_supplier/views/product_supplier_view.xml [hunk 3]
```diff
@@ -268,17 +292,22 @@
...
+        <a href="/my/supplier/product/new" class="btn btn-primary pull-right">
+            <i class="fa fa-plus"></i>
+        </a>
+      </span>
+    </div>
 </template>
 
 </data>
-</openerp>
+</openerp>
```
Comment (yajo) [inline website_product_supplier/views/product_supplier_view.xml]: Leave the line feed please.

**Comment by yajo** [comment]
Bots will fail until https://github.com/OCA/website/pull/152 is merged and this PR rebased.

**Comment by sergio-teruel** [comment]
@Yajo Changes done

**Comment by antespi** [comment]
LGTM :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by rafaelbn** [comment]
Thanks :+1:

**Pull request merged**